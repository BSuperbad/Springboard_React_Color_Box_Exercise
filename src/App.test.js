import { render } from '@testing-library/react';
import App from './App';

it("renders App without crashing", function () {
    render(<App />);
});

it("matches App snapshot", function() {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
});
