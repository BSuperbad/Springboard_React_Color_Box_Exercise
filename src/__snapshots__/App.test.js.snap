// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches App snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <form>
        <label
          for="color"
        >
          Color: 
        </label>
        <input
          id="color"
          name="color"
          placeholder="Box Color"
          type="text"
          value=""
        />
        <label
          for="width"
        >
          Width: 
        </label>
        <input
          id="width"
          max="500"
          min="100"
          name="width"
          type="range"
          value=""
        />
        <label
          for="height"
        >
          Height: 
        </label>
        <input
          id="height"
          max="500"
          min="100"
          name="height"
          type="range"
          value=""
        />
        <button
          type="submit"
        >
          Add Box
        </button>
      </form>
      <div>
        <div
          style="background-color: blue; width: 100px; height: 50px;"
        >
          <button>
            X
          </button>
        </div>
        <div
          style="background-color: red; width: 100px; height: 250px;"
        >
          <button>
            X
          </button>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
